.travis.yml

```diff
@@ -12,7 +12,8 @@ install:
 - npm install
 - bower install
 after_success:
-- sed -i -- 's/SF:ember-frost-sidebar\/\(.*\)/SF:addon\/\1.js/' coverage/lcov.info && rm -f coverage/lcov.info--
```

Comment:
ran this command again: curl -O http://leonardo.cyanoptics.com:3333/ciena-frost/ember-frost-sidebar/.travis.yml

But my content is still not some parts. For example, it does not the following part.
script:
- npm run lint
- ember try:one $EMBER_TRY_SCENARIO --- ember test